File: server/java/src/main/java/com/onlinepayments/example/api/models/hostedcheckout/CreateHostedCheckoutResponse.java
package com.onlinepayments.example.api.models.hostedcheckout;

import com.onlinepayments.example.service.domain.common.enums.Currency;

import java.math.BigDecimal;

public record CreateHostedCheckoutResponse(
        String hostedCheckoutId,
        String redirectUrl,
        String returnMAC,
        BigDecimal amount,
        Currency currency
) { }

File: server/dotnet/sdk-dotnet-example/Presentation/Models/CreateHostedCheckouts/CreateHostedCheckoutResponse.cs
using Business.Domain.Common.Enums;

namespace Presentation.Models.CreateHostedCheckouts;

public class CreateHostedCheckoutResponse
{
    public string? HostedCheckoutId { get; init; }
    
    public string? RedirectUrl { get; init; }
    
    public string? ReturnMAC { get; init; }
    
    public decimal? Amount { get; init; }
    
    public Currency? Currency { get; init; }
}